CRR/ReplicationStatusUpdater.js

```diff
@@ -114,16 +114,19 @@ class ReplicationStatusUpdater {
      * Determines if an object should be updated based on its replication metadata properties.
      * @private
      * @param {ObjectMD} objMD - The metadata of the object.
+     * @param {string} site - The destination site name.
      * @returns {boolean} True if the object should be updated.
      */
-    _objectShouldBeUpdated(objMD) {
+    _objectShouldBeUpdated(objMD, site) {
         return this.replicationStatusToProcess.some(filter => {
             if (filter === 'NEW') {
+                // Either site specific replication info is missing
+                // or are initialized with empty fields.
                 return (!objMD.getReplicationInfo()
-                    || objMD.getReplicationInfo().status === '');
+                    || !objMD.getReplicationSiteStatus(site));
             }
             return (objMD.getReplicationInfo()
-                && objMD.getReplicationInfo().status === filter);
+                && objMD.getReplicationSiteStatus(site) === filter);
```

Comment:
is this true as well for CRR (in s3c) ?

esp. we should check that there is no "legacy" data format where we would not have the per-site status, and where we should rely only on the global state...

Reply:
@nicolas2bert @jonathan-gramain what do you think?